solutions/usaco-988.mdx

```diff
@@ -7,14 +7,83 @@ author: Ananth Kashyap
 
 [Official Analysis (C++)](http://www.usaco.org/current/data/sol_photo_bronze_jan20.html)
 
+Since $N \leq 1000$, we can simulate every ordering and pick the first valid permutation (since that would be the lexicographical minimum).
+
 ## Implementation
 
-**Time Complexity:** $\mathcal{O}(N^3)$
 
 <LanguageSection>
 
+<CPPSection>
+
+**Time Complexity:** $\mathcal{O}(N^2)$
+
+```cpp 
+#include <iostream>
+#include <vector>
+#include <algorithm>
+#include <cstdio>
+
+using namespace std;
+
+int main() {
+	freopen("photo.in", "r", stdin);
+	freopen("photo.out", "w", stdout);
+	int n; cin >> n;
+	int order[1000];
```

Comment:
```suggestion
	vector<int> order(n);
```

Reply:
instead of `order[i] = "elem"`, should it be changed to `order[i].pb('elem')`?

Reply:
or does it still work

Reply:
wait nvm it already has a size